pkg/dao/types/status/walker.go

```diff
@@ -215,6 +215,8 @@ func getGeneralDecide[T ~string](step T) Decide {
 var replacements = map[string]struct {
 	T, E, D string
 }{
+	"Running":     {"Running", "Failed", "Completed"},
+	"Pending":     {"Pending", "Failed", "Pended"},
```

Comment:
What status will display while the Pending status is True? Pended -> Pending, please double check here?